flexget/plugins/metainfo/series.py

```diff
@@ -43,7 +43,15 @@ def guess_entry(self, entry, allow_seasonless=False, config=None):
         identified_by = 'auto'
         if config and 'identified_by' in config:
             identified_by = config['identified_by']
-        parsed = get_plugin_by_name('parsing').instance.parse_series(data=entry['title'], identified_by=identified_by,
+        if entry.get('series_name'):
+            title = entry['series_name']
+            if entry.get('series_year'):
+                title += ' ({})'.format(entry['series_year'])
+            if entry.get('series_id'):
+                title += ' {}'.format(entry['series_id'])
```

Comment:
what's the use case for this? i don't remember what's series_id...

Reply:
I think it's the value of the ep identifier